.pa11yci.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+
+let options = {
+  defaults: {
+    // 'Section508', 'WCAG2A', 'WCAG2AA', or 'WCAG2AAA'
+    standard: 'WCAG2AA',
+  },
+  urls: []
+};
+
+// This recursively scans the docs folder to find the URLs to test with Pa11y
```

Comment:
thought this would be better than having to manually update this file for every new component.

Reply:
Yeah, this is a great idea. I've been thinking if I should setup something to run this against a deployed site, but getting it up locally is a great first step.